app/_how-tos/connect-azure-devops-to-the-konnect-catalog-with-the-konnect-ui.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Connect Azure DevOps repositories to Catalog with the {{site.konnect_short_name}} UI
+content_type: how_to
+description: Learn how to connect Azure DevOps repositories to your {{site.konnect_catalog}} services in {{site.konnect_short_name}} using the UI.
+products:
+  - catalog
+works_on:
+  - konnect
+tags:
+  - integrations
+search_aliases:
+  - devops
+  - azure repos
+  - service catalog
+related_resources:
+  - text: "{{site.konnect_catalog}}"
+    url: /catalog/
+  - text: Integrations
+    url: /catalog/integrations/
+  - text: Azure DevOps reference
+    url: /catalog/integrations/azure-devops/
+  - text: "Connect Azure DevOps repositories to Catalog with the Konnect API"
+    url: /how-to/connect-azure-devops-with-konnect-api/
+automated_tests: false
+tldr:
+  q: How do I connect Azure DevOps repositories to {{site.konnect_short_name}}?
+  a: Install the Azure DevOps integration in {{site.konnect_short_name}}, authorize it with a Personal Access Token (PAT) that has Code:Read access, then map an Azure DevOps repository to your {{site.konnect_catalog}} service.
+prereqs:
+  inline:
+    - title: "{{site.konnect_short_name}} roles"
+      include_content: prereqs/service-catalog-integration-role
+      icon_url: /assets/icons/kogo-white.svg
+    - title: Create and configure an Azure account
+      content: |
+        You need to configure the following in Azure DevOps:
+        - An [Azure DevOps account](https://azure.microsoft.com/en-gb/pricing/purchase-options/azure-account?icid=devops).
+        - An [Azure DevOps personal access token (PAT)](https://learn.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate?view=azure-devops&tabs=Windows) with `Code: Read` permission.
+        
+        {:.warning}
+        > Your PAT can be created with an expiration period of your choice, up to a maximum of one year. Make sure to renew the PAT before it expires to avoid interruptions.
+---
+
+## Configure the Azure DevOps integration
+
+Before you can discover Azure DevOps repositories in {{site.konnect_catalog}}, you must configure the integration:
+
+1. In the {{site.konnect_short_name}} sidebar, click **Catalog**.
+1. In the {{site.konnect_catalog}} sidebar, click **Integrations**.
+1. Click **Azure DevOps**.
+1. Click **Add Azure DevOps instance**.
```

Comment:
This says "Add Instance" for me 
<img width="151" height="56" alt="image" src="https://github.com/user-attachments/assets/68cb4e16-543e-4b78-94e0-ca7768cb2669" />

Does it say **Add Azur DevOps instance** If there are no other instances?